cart/questions.py

```diff
@@ -287,5 +292,10 @@ def process_questions(root, questions):
         return root
 
     def write_to_file(self, file):
```

Comment:
This override is not necessary.

Reply:
do you mean to completely remove `write_to_file`? I am not sure where that is used. Probably only CartEstimateJob.

Reply:
As `CMUCartQuestions` inherits from `BasicCartQuestions` and the same implementation is already present in `BasicCartQuestions`, this function `write_to_file` will override the exact same function `write_to_file` from `BasicCartQuestions` with exactly the same implementation.

Therefore, deleting the function here will lead to the exact same behavior.

Reply:
But the function `write_to_xml` belongs to `PythonCartQuestions` and not to `CMUCartQuestions`. And `PythonCartQuestions` does not inherit from `BasicCartQuestions`